### PR TITLE
slack logging

### PR DESCRIPTION
**Pull request by minsulee2**
- Now enable slack logging
- Add error object

### CODE REVIEW

File: slack.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const slack = new Slack();
+slack.setWebhook(`https://hooks.slack.com/services/${SLACK_WEBHOOK_TOKEN}`);
+
+const sendSlackMsg = (message, errObject) => {
+  try {
+    slack.webhook({
+      channel: SLACK_CHANNEL_NAME,
+      icon_emoji: ':dragon:',
+      username: 'AINFT dragon',
```
Comment (jiyoung-an) [inline slack.js:13]: 🐉

File: handlers/error.js [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
 const { REST_MODE } = require('../env');
+const { sendSlackMsg } = require('../slack');
 
 class ErrorHandler {
   static sendErrorResponse(err, req, res, next) {
     const statusCode = err.status || 500;
     const redirectPath = err.redirectPath || null;
     // FIXME(minsu): This will be removed when it is stablized. This console log is for debugging on the container.
-    console.log(`error: ${err.message}`);
+    console.log(`error: ${err.message} with object(${err.errorOriginObject})`);
```
Comment (jiyoung-an) [inline handlers/error.js:11]: 👍

**Review by jiyoung-an — APPROVED** [review]
LGTM, thank you for quickly creating a monitoring channel! 😇

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by minsulee2** [comment]
Thanks! Merge and deploy now :)

**Pull request merged**